espflash/src/cli/mod.rs

```diff
@@ -453,6 +453,7 @@ pub fn connect(
         !no_verify,
         !no_skip,
         args.chip,
+        args.baud.or(config.project_config.baudrate),
```

Comment:
This adds a weird arg to the flasher, but the other alternative was to have 2 getters in connection.